### DISCUSSION TITLE
Organisation search not working with search api

### DISCUSSION THREAD

**Discussion by nehaghub** [community/community#128138] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Hi Team,

Below is my code for code search but no result is coming for my organization.
How can i check what is the issue. Can you please help me for the same.
`

import requests
import json

url = "https://api.github.com/search/code?q=org:myorgname+iics&type=code
headers = {
  'Authorization': 'Token my_token'
}

response = requests.request("GET", url,headers=headers)
print(response)
print(json.dumps(response.json(), indent=4))`

Thanks in Advance.
Neha Goel

**Comment by buu-huu** [comment, marked as answer]
Hi Neha,

you have an error in your authorization header. Try it like this:

``` python
headers = {
    "Authorization": "token <YOUR_TOKEN>"
}
```

Also make sure, that your token has the appropriate OAuth scopes / permissions.

**Reply by grandmas-favorite** [reply]
Nice catch @buu-huu !

**Reply by nehaghub** [reply]
Thank you so much @buu-huu. My issue resolved using this. :)

**Reply by buu-huu** [reply]
So glad to hear! Thanks for marking it as answer :)

**Comment by Www11rocking** [comment]
Hi
